.github/workflows/auto-updates.yaml

```diff
@@ -63,17 +59,13 @@ jobs:
     # This should just be "after", but we don't want the 2 jobs to push at the same time
     needs: update-po
     runs-on: ubuntu-latest
-    container: ubuntu:rolling
```

Comment:
Just a remark: the containers were needed at the time to get some fixes we needed in libsmclient-dev to generate what we needed. Indeed, not needed anymore, thanks for cleaning it up!